package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "summary-menu",
+  "version": "1.0.0",
+  "description": "the summary and menu components for Open table clone coding",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev:react": "webpack -d --watch"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/sigsa-io/summary-menu.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/sigsa-io/summary-menu/issues"
+  },
+  "homepage": "https://github.com/sigsa-io/summary-menu#readme",
+  "devDependencies": {
+    "@babel/preset-env": "^7.4.5",
+    "@babel/preset-react": "^7.0.0",
+    "babel-core": "^6.26.3",
+    "babel-loader": "^8.0.6",
+    "babelrc": "0.0.2",
+    "jest": "^24.8.0",
+    "webpack": "^4.35.2",
+    "webpack-cli": "^3.3.5"
+  },
+  "dependencies": {
+    "@babel/core": "^7.4.5",
```

Comment:
@cminho921 
@LittleJennie 

Why is this a dependency and the others are dev dependencies?